admin-dev/themes/new-theme/js/pages/product/components/create-product-modal.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+import Router from '@components/router';
+import ProductMap from '@pages/product/product-map';
+import {FormIframeModal} from '@components/modal';
+
+export default class CreateProductModal {
+  private router: Router;
+
+  constructor() {
+    this.router = new Router();
+    this.init();
+  }
+
+  private init(): void {
+    $(ProductMap.create.newProductButton).on('click', (event: JQuery.ClickEvent) => {
+      event.preventDefault();
+      const $link = $(event.target);
+      const linkUrl = `${$link.prop('href')}&liteDisplaying=1`;
```

Comment:
why not adding the query param to the href prop directly ?

Reply:
So that the button link is still valid You can open the create page in another tab if you want in which case it must be displayed with the full layout
The lite display is only pertinent when displayed in the modal